docs/source/models/concept_models/vivarium_mncnh_portfolio/concept_model.rst

```diff
@@ -918,6 +918,61 @@ Default stratifications to all observers should include scenario and input draw.
       * ACS eligibility (dichotomous, 'eligible' if believed gestational age is between 26-33 weeks, 'not eligible' if gestational age is outside of this range)
       * ACS coverage
     - 
+  * - 8. Neonatal all-cause mortality risk
+    - * Child sex
+      * Child age group
+      * Delivery facility type
+      * CPAP availability
+      * Antibiotics availability
+      * Probiotics availability
+      * Corticosteroid coverage
+      * ACS eligibility (dichotomous, 'eligible' if believed gestational age is between 26-33 weeks, 'not eligible' if gestational age is outside of this range)
+      * Preterm status (dichotomous at 37 weeks)
+    - Observe the following statistics about the ACMRisk pipeline used to sample for deaths:
+
+      * The number of values (equal to the number of living simulants)
+      * The sum of the values
+      * The sum of the squares of the values
+
+      All of these quantities can aggregate across seeds in the normal way (summation).
+  * - 9. Neonatal cause-specific mortality risks (per cause)
+    - * Child sex
+      * Child age group
+      * Delivery facility type
+      * CPAP availability
+      * Antibiotics availability
+      * Probiotics availability
+      * Corticosteroid coverage
+      * ACS eligibility (dichotomous, 'eligible' if believed gestational age is between 26-33 weeks, 'not eligible' if gestational age is outside of this range)
+      * Preterm status (dichotomous at 37 weeks)
+    - Observe the following statistics about the CSMRisk pipelines just *before* they are normalized 
```

Comment:
I think a little more expansion on exactly what quantity this is would be helpful for interpretation - maybe linking to the relevant quantity on the [neonatal mortality document](https://vivarium-research.readthedocs.io/en/latest/models/causes/neonatal/index.html) and/or the quantity in the sim implementation?

I think this also applies to the parameter above and below as well!